### DISCUSSION TITLE
Welcome to the Cloud category for Universe 2022!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/events#13] [Cloud]
☁️  What does productivity look like when developers are given a powerful platform in the cloud? Scary fast. Learn how different industries best use software in the cloud to enable collaboration, accelerate productivity, and transform their businesses. For questions about this space check out the [FAQ](https://github.com/community/events/discussions/1)  and don't forget you can [tune in live](http://watch.githubuniverse.com/). Let us know what sessions you're attending or loving below ⬇️ !

**Comment by WillOfHeaven** [comment]
This is my first time attending GitHub Universe , as a Student and a new kid in the block of Computer Science I am Stoked to be a part of this Event !!

**Reply by ossanna16** [reply]
Welcome @WillOfHeaven! We are so glad to have you here! Happy Universe :octocat:
